snuba/query/processors/granularity_processor.py

```diff
@@ -64,11 +65,28 @@ class MappedGranularityProcessor(QueryProcessor):
 
     def __init__(
         self,
-        accepted_granularities: Sequence[GranularityMapping],
+        accepted_granularities: Union[
+            Sequence[GranularityMapping], Sequence[Sequence[int]]
+        ],
         default_granularity_enum: int,
     ):
+        # handle YAML not knowing about the GranularityMapping NamedTuple
```

Comment:
We may want to consider having all query processors/validators take basic python types (int, str, Sequence[str], Sequence[int], Mapping[str,str] etc.) to avoid this kind of stuff

Reply:
A quick survey of the codebase says we're mostly not using custom types in QueryProcessor subclass initializers

Reply:
I'm for this

Reply:
That makes sense to me too.